src/metric-engine/src/error.rs

```diff
@@ -182,7 +196,10 @@ impl ErrorExt for Error {
             | MissingRegionOption { .. }
             | ConflictRegionOption { .. }
             | ColumnTypeMismatch { .. }
-            | PhysicalRegionBusy { .. } => StatusCode::InvalidArguments,
+            | PhysicalRegionBusy { .. }
```

Comment:
The status code of `PhysicalRegionBusy` must be `RegionBusy`.

https://greptimedb.rs/common_error/status_code/enum.StatusCode.html#variant.RegionBusy

Reply:
"busy" here means there are still some logical regions using it. Like dropping a physical region when its logical regions are not dropped first. Hence invalid argument